src/components/AppProvider/AppProvider.tsx

```diff
@@ -95,25 +98,27 @@ export class AppProvider extends React.Component<AppProviderProps, State> {
   }
 
   render() {
-    const {theme = {logo: null}, children} = this.props;
+    const {theme = {logo: null}, settings, children} = this.props;
```

Comment:
Do we know if children will receive an update when this changes?

Reply:
Yep, looks like it.